shells/firefox/build.js

```diff
@@ -25,13 +25,13 @@ const main = async () => {
   console.log(chalk.gray('\n# Go to the unpacked directory:'));
   console.log(`cd ${unpackedPath}`);
   console.log(chalk.gray('\n# And launch Firefox with the extension enabled:'));
-  console.log('web-ext run');
+  console.log('yarn run start:firefox');
```

Comment:
Line 23 above already mentions the preferred testing route. These instructions just list out the manual steps for the curious.